### ISSUE TITLE
Adding a new host for a user with an underscore in their name causes a permissions error

### ISSUE THREAD

**Issue by sbenskin** [sequelpro/sequelpro#2534]
Adding a new host for a user with an underscore in their name causes a permissions error.

I think it's related to assigning schema specific permissions only as well

It also inserts the row into mysql.db with the underscore escape

**Comment by dmoagx** [comment]
> Adding a new host for a user with an underscore in their name causes a permissions error.

That is *not* a valid issue description for a bug report!

> It also inserts the row into mysql.db with the underscore escape

If you are talking about the database name, then yes, that's how it is supposed to work:
https://dev.mysql.com/doc/refman/5.7/en/request-access.html

**Comment by sbenskin** [comment]
A chance to reply before closing this ticket would have been nice.

**Comment by dmoagx** [comment]
Well, you could reopen it, when you have relevant details.

**Comment by sbenskin** [comment]
Sorry that you found my report didn't contain enough information.  I've since found out that sequel pro runs a flush privileges after adding the user, which my user didn't have access to do, so that was my problem.

It would be helpful when seeing the error for access denied, to see the SQL it is trying to run at that time. I had to dig into the console to find it.  Even better would be to interpret which permission is missing, but that would probably too much work vs the benefit.

Just a nice to have, I'm sure you have more important issues to work on.
